Code/GraphMol/ChemReactions/ReactionRunner.cpp

```diff
@@ -287,33 +287,31 @@ struct ReactantProductAtomMapping {
 ReactantProductAtomMapping *getAtomMappingsReactantProduct(
     const MatchVectType &match, const ROMol &reactantTemplate,
     RWMOL_SPTR product, unsigned numReactAtoms) {
-  ReactantProductAtomMapping *mapping =
+  auto *mapping =
       new ReactantProductAtomMapping(numReactAtoms);
-  for (unsigned int i = 0; i < match.size(); i++) {
-    const Atom *templateAtom = reactantTemplate.getAtomWithIdx(match[i].first);
+  for (const auto & i : match) {
```

Comment:
I'm not sure this is validating the conversion.  In other cases is looks like it has some knowledge of english in this case there is no english.  I think i as an index was descriptive in the past but here we need to actually change the style of the vector naming:

```
for (const auto &match : matches)
```

And this requires a change outside the scope of the conversion (but a good one nonetheless)